buildtest/cli/compilers.py

```diff
@@ -127,6 +128,7 @@ def compiler_find(configuration, modulepath=None, detailed=None, update=None):
         modulepath (List, optional): An instance of List, a list of directories to search for modules via MODULEPATH to detect compilers
         detailed (bool, optional): An instance of bool, flag for printing a detailed report.
         update (bool, optional): An instance of bool, flag for updating configuration file with new compilers
+        filepath (str, optional): An instance of str, an option to specify an alternative filepath where configuration file will be run
```

Comment:
I think we can remove `An instance of` from all the variables. 

Also can you change `run` -> `written`. Since the configuration file is written.